src/v0/destinations/tiktok_ads/transform.js

```diff
@@ -152,10 +152,11 @@ const trackResponseBuilder = async (message, { Config }) => {
         });
       }
     });
-  } else {
-    event = eventNameMapping[event];
-    responseList.push(getTrackResponse(message, Config, event));
+    return responseList;
   }
+  event = eventNameMapping[event] || event;
```

Comment:
@anantjain45823 can we add the doc link which is in support that we can send custom events. I see we have no doc refs added for track events as a whole as well.

Reply:
Done.

Reply:
Added the doc hence resolving